### PR TITLE
[integration] Add PR template and contributing guidelines

### PR DESCRIPTION
**Pull request by petricm**
As discussed today this is the PR template that is the basis for the auto generation of release notes.
This will appear in all future PRs, users only need to fill it out.

I have also added some basic contributing guidelines. A link to this .md file will appear when you want to start a new PR.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+# Contribution Guidelines for DIRAC
+
+
+## Pull Requests
+
+When making a `Pull Request` please explain what and why things were
+changed. Please include a short description in the
+BEGINRELEASENOTES/ENDRELEASENOTES block, which will appear in the
+doc/ReleaseNotes.md, once a new tag is made.
```
Comment (andresailer) [inline .github/CONTRIBUTING.md]: It will go to relase.notes

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+in your pull-request and commit messages with `#[issue-id]`.
+
+## Coding Conventions
+
+ * Your code should not introduce any new pylint warnings, and fix as many existing warnings as possible
+
+## Git workflow
+
+For an explanation of the git(hub) workflow please see
+[here](https://github.com/andresailer/tutorial#working-updating-pushing).  We
```
Comment (andresailer) [inline .github/CONTRIBUTING.md]: Dirac is using a different workflow, point to the dirac docs

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Issue tracking
+
+Use the GitHub issue tracker. Reference the issues that you are working on.
+If you notice an issue, consider first creating an issue and then refering to it
+in your pull-request and commit messages with `#[issue-id]`.
+
+## Coding Conventions
+
+ * Your code should not introduce any new pylint warnings, and fix as many existing warnings as possible
```
Comment (andresailer) [inline .github/CONTRIBUTING.md]: Point to the dirac coding conventions in the docs

File: .github/PULL_REQUEST_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+
+BEGINRELEASENOTES
+- Thank you for writing the text to appear in the release notes. It will show up
```
Comment (andresailer) [inline .github/PULL_REQUEST_TEMPLATE]: Should show some example as they appear in the release.notes
```
*SomeSystem
NEW/CHANGE/FIX: explanation
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Pull Requests
+
+When making a `Pull Request` please explain what and why things were
+changed. Please include a short description in the
+BEGINRELEASENOTES/ENDRELEASENOTES block, which will appear in the
+doc/ReleaseNotes.md, once a new tag is made.
+
+We are happy if you create pull-requests also if you feature is not ready, yet.
+Please mark them as such by adding `[WIP]` to the start of the title. The purpose
```
Comment (andresailer) [inline .github/CONTRIBUTING.md]: Also add that people should put `[targetRelease]` in the title of the PR

**Comment by fstagni** [comment]
Thanks about this. Would you mind also update/cross-reference this guide? http://dirac.readthedocs.io/en/integration/DeveloperGuide/DevelopmentModel/ContributingCode/index.html

**Pull request merged**